modules/electrophysiology_browser/jsx/components/electrophysiology_session_panels.js

```diff
@@ -18,7 +19,9 @@ class FilePanel extends Component {
     super(props);
     this.state = {
       data: this.props.data,
+      id: this.props.fileId,
```

Comment:
For clarity, can this key and the props be renamed for something more explicit (like physiologicalFileID?)